### PR TITLE
Completes Week 3 Walkthrough #2

### PR DESCRIPTION
**Pull request by mmurray22**
Completes the second walkthrough of Week 3

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -70,3 +70,11 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+async function fetchHelloWorld() {
+	document.getElementById("inner-content").style.display = "block";
```
Comment (ccondit) [inline portfolio/src/main/webapp/script.js]: nit - this seems unrelated to this method (at least to the casual reader). Also, since this method is invoked on page load, would it make sense to just style this element with css?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -70,3 +70,11 @@ function contact() {
...
+	const response = await fetch('/data');
+	const hello = await response.text();
```
Comment (ccondit) [inline portfolio/src/main/webapp/script.js]: nit - can you choose a more descriptive variable name? `helloMessage` or something like that?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -70,3 +70,11 @@ function contact() {
...
+	document.getElementById("my-info").innerText = hello;
```
Comment (ccondit) [inline portfolio/src/main/webapp/script.js]: nit - per style guide, use [single quotes](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes) instead of double quotes when quoting strings.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -70,3 +70,11 @@ function contact() {
...
+	console.log(`Fetch Hello World!`);
```
Comment (ccondit) [inline portfolio/src/main/webapp/script.js]: uber nit - i don't know this for sure, but based on the style guide for quotes I think we should be using single quotes where possible and template strings when the string contains a single quote.

**Review by ccondit — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/script.js]: thanks! will do!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/script.js]: thanks! will do!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/script.js]: thanks! will do!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/script.js]: thanks! will do!

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -70,3 +70,10 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+async function fetchHelloWorld() {
+	const response = await fetch('/data');
+	const helloMessage = await response.text();
+	document.getElementById('my-info').innerText = hello;
```
Comment (ccondit) [inline portfolio/src/main/webapp/script.js]:
```suggestion
	document.getElementById('my-info').innerText = helloMessage;
```

**Review by ccondit — APPROVED** [review]
LGTM!

**Review by kaveh096 — APPROVED** [review]
